src/lib/core/overlay/scroll/scroll-dispatcher.ts

```diff
@@ -69,10 +71,12 @@ export class ScrollDispatcher {
     this._scrolledCount++;
 
     if (!this._globalSubscription) {
-      this._globalSubscription = Observable.merge(
-        Observable.fromEvent(window.document, 'scroll'),
-        Observable.fromEvent(window, 'resize')
-      ).subscribe(() => this._notify());
+      this._globalSubscription = this._ngZone.runOutsideAngular(() => {
```

Comment:
Add a comment explaining why we need to escape from the Angular zone here (and in the directive)?
(I generally like to document the "why" every time we do this)